pkg/virt-controller/watch/migration.go

```diff
@@ -1407,6 +1400,43 @@ func (c *MigrationController) getNodeForVMI(vmi *virtv1.VirtualMachineInstance)
 	return node, nil
 }
 
+func (c *MigrationController) alertIfHostModelIsUnschedulable(vmi *virtv1.VirtualMachineInstance, targetPod *k8sv1.Pod) {
+	const warningMsgPattern = "Migration cannot proceed since no node is suitable to run the required CPU model / required features: %v"
+	fittingNodeFound := false
+
+	if cpu := vmi.Spec.Domain.CPU; cpu == nil || cpu.Model != virtv1.CPUModeHostModel || !isPodPending(targetPod) {
```

Comment:
Because if the scenario is that the Pod cannot be scheduled due to no node supporting its model (or in other words - no node has labels that fit to its nodeSelector) it will be in pending phase. If, for example, the pod is already scheduled or scheduling - there is no point of making these checks.